site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Progress in the Privacy Sandbox (December 2021)
+description: >
+  Looking back at stats from 2021 and some resolutions for tackling user-agent and cookies in 2022.
+layout: 'layouts/blog-post.njk'
+date: 2022-01-07
+authors:
+  - rowan_m
+hero: 'image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/Ut0IykTaEFBuOA5EHqbg.png'
+alt: >
+  Privacy Sandbox logo
+tags:
+  - progress-in-the-privacy-sandbox
+  - privacy
+---
+
+Welcome to the 2021 wrap-up edition of **[Progress in the Privacy
+Sandbox](/tags/progress-in-the-privacy-sandbox/)** tracking the milestones on
+the path to phasing out third-party cookies in Chrome and working towards a more
+private web. Normally, we share an overview of updates to the **[Privacy Sandbox
+timeline](https://privacysandbox.com/timeline/)** along with news from across
+the project, but December was a quiet month. Instead, we'll look back at some of
+the activities over 2021 and provide a few potential resolutions you can make
+now we are in the new year.
+
+## Chromium development process
+
+For a new web platform feature to land in Chromium, it needs to go through the
+[project's open development
+process](https://www.chromium.org/blink/launching-features). In 2021, the
+Privacy Sandbox team produced:
+
+<figure>
+  <table style="font-size: 140%; font-weight: bold; font-variant-numeric: tabular-nums;">
+  <tr><td>🥚</td><td>Intent to Prototype </td>     <td>19</td></tr>
+  <tr><td>🧪</td><td>Intent to Experiment</td><td>&nbsp;7</td></tr>
+  <tr><td>🚀</td><td>Intent to Ship      </td>     <td>11</td></tr>
+  </table>
+  <figcaption>
+    <a href="https://docs.google.com/spreadsheets/d/e/2PACX-1vR-5YYZ-lxG9xelJFCYM2yajUEWUNKZkiXcWX9BYi8dSUV_5Xrhd5u-iyNNf4w8NAVwrSrVWjeNrC-i/pubhtml#">Source</a>
+  </figcaption>
+</figure>
+
+Each milestone is also an invitation for the wider web ecosystem to contribute.
+
+An **Intent to Prototype** is the first checkpoint, where we invite discussion
+and early experimentation. This means a proposal will be available on GitHub,
+where **you can ask a question by creating an Issue** or **[join discussions and
+presentations in standards
+groups](/docs/privacy-sandbox/cds21-update/#discussion)** such as the W3C and
+IETF. This is also where coding begins, which means you can **expect the
+prototype functionality to be made available for developer testing behind a
+[feature
+flag](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)**.
+Initial feedback is critical for validating and iterating on proposals.
+
+An **Intent to Experiment** is an optional step if we’d like to request an
+origin trial. Developers **can [sign up for the origin
+trial](/blog/origin-trials/)** of a feature, and then **test  it on your
+production site**. It's called an experiment because we have specific aspects of
```

Comment:
May this should be:

> Developers **can sign up for the origin trial** of a feature, and then **test it in production**.

Removing the "you" so it's not third-person and second-person in the same sentence.